packages/neon_framework/test/weather_status_bloc_test.dart

```diff
@@ -211,4 +211,11 @@ void main() {
       ..setLocation('Hamburg')
       ..setLocation('Berlin');
   });
+
+  test('Do not refresh if location has no data', () async {
+    bloc.location.add(Result.loading());
+    bloc.location.add(Result.success(null));
```

Comment:
Do not modify the subject directly, you need to provide some mock responses that trigger this case.